packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -162,6 +162,11 @@ class MethodChannelCamera extends CameraPlatform {
         .whereType<DeviceOrientationChangedEvent>();
   }
 
+  @override
+  Stream<CameraTimeLimitReachedEvent> onCameraTimeLimitReached(int cameraId) {
```

Comment:
I think it would be better to rename this to `onVideoRecordingTimeLimitReached`, so it is clear this is directly related to the video recording feature. Of course if you accept my earlier comment about also emitting the recorded video on this stream when the user calls `stopVideoRecording` we should rename it to something more generic like `onVideoRecorded`.